### ISSUE TITLE
Cleanup translations

### ISSUE THREAD

**Issue by rijkvanzanten** [directus/next#230]
- [ ] Figure out ideal file structure*
- [ ] Move from JSON to YAML
- [ ] Check integration with Crowdin
- [ ] Remove unused translation strings
- [ ] Group translations where useful / applicable

\* We can simplify the translation file by splitting it up into multiple scoped files (eg everything related to "settings" in one file), _or_ we can instead move everything into the same file, and use nested groups for this scoping.

I believe Crowdin supports searching through the whole codebase for the translation files based on a wildcard, so we could also consider putting translations next to where they're used (eg move all settings translations to the settings module)

**Comment by Nitwel** [comment]
My take on this would be to have a translation file for
- /src/components
- /src/composables (if needed)
- /src/directives
- /src/displays
- /src/interfaces
- /src/layouts
- /src/modules
- /src/views

If we want we can break this further down if wanted because everything will be organized properly.
Also, if we decide to move everything back into a single folder, it's just a matter of moving files and not restructuring them.

**Comment by rijkvanzanten** [comment]
Not too sure about that, all keys are global by nature, and a lot will be shared between these different types (eg we don't want things like "done" or "user" translated in 10 different places)

**Comment by Nitwel** [comment]
That is true. I had some more thoughts on this and would say lets discuss this in discord.
